### PR TITLE
Gherkin update

### PR DESCRIPTION
**Pull request by mlvandijk**
Merge feature-introduction into gherkin-reference
Update links
Merge info from https://github.com/cucumber/cucumber/pull/68 into introduction

### CODE REVIEW

File: content/gherkin/gherkin-reference.md [hunk 1]
```diff
@@ -13,9 +13,9 @@ Cucumber executes your `.feature` files, and those files contain executable spec
 written in a language called Gherkin.
 
 Gherkin is plain-text English (or one of 60+ other languages) with a little extra structure.
-Gherkin is designed to be easy to learn by non-programmers, yet structured enough to
+Gherkin is designed to be easy for people to read and write, yet structured enough to
 allow concise description of examples to illustrate business rules in most real-world
-domains.
+domains. It is structured enough for a tool like Cucumber to process.
```
Comment (plaindocs) [inline content/gherkin/gherkin-reference.md]: How about

> Gherkin is designed to be easy for people to read and write, yet structured enough to
 to allow for automated processing and concise description of business rules.

File: content/gherkin/gherkin-reference.md [hunk 2]
```diff
@@ -49,30 +49,44 @@ There are a few extra keywords as well:
...
 of a software Feature, and to group related Scenarios.
 
-A Feature has three basic elements---the `Feature:` keyword, a *name* (on the same line)
-and an optional (but highly recommended) *description*, which can span multiple lines.
+The `.feature` file starts with the keyword **Feature**, a *name* (on the same line),
+followed by an optional (but highly recommended) free format *description* which can span multiple lines.
+The free format description ends when the first scenario starts.
 
-Cucumber does not care about the name or the description. Their purpose is simply to provide a place for you to document important aspects of the Feature, such as a brief explanation and a list of business rules (general acceptance criteria).
+Cucumber does not care about the name or the description.
```
Comment (plaindocs) [inline content/gherkin/gherkin-reference.md]: Slightly wary of "does not care" here. Can we reword?

Also remove "simply" ?

**Comment by plaindocs** [comment]
Otherwise looks great.

File: content/gherkin/gherkin-reference.md (hunk 1 above)
Comment (mlvandijk) [reply content/gherkin/gherkin-reference.md]: Much better, thanks!

File: content/gherkin/gherkin-reference.md (hunk 2 above)
Comment (mlvandijk) [reply content/gherkin/gherkin-reference.md]: Changed to: "The name and the description have no special meaning to Cucumber." (and removed "simply").
Let me know if anyone has a better idea

**Pull request merged**